### ISSUE TITLE
 PTN000172598 not all GO:0005200 | structural constituent of cytoskeleton 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5398]
* **PTHR ID & PTN node:**

GO:0005200 | structural constituent of cytoskeleton | IBA with MGI:107814 , FBgn0284243 , CAL0000176403 , P18695 , PTN000172598 , P68363 , P24633 , Q13509 , A0A1D8PTV4 , S000001857 , DDB_G0287689 , DDB_G0269196 , FBgn0003886 , S000004593 , MGI:107812 , FBgn0052396 , S000004202

* **Sequences with problematic annotation (ID + gene/protein name):**

All. gamma tubulins
https://www.pombase.org/gene/SPBC32F12.04 gtb1 (gamma tubulin)
https://www.uniprot.org/uniprotkb/P23258/entry human TUBG1
https://www.uniprot.org/uniprotkb/Q9NRH3/entry human TUBG2
https://www.yeastgenome.org/locus/S000004202#go

This is propagation from normal tubulin family

pombe has EXP to 
GO:0140490 | microtubule nucleator activity | EXP | Masuda H et al. (1996)
that could be PAINTED instead

[GO:0140490](https://www.pombase.org/term/GO:0140490)	
[microtubule nucleator activity](https://www.pombase.org/term/GO:0140490)
[EXP](http://wiki.geneontology.org/index.php/Inferred_from_Experiment_(EXP))	[Masuda H et al. (1996)](https://www.pombase.org/reference/PMID:8834801)

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by ValWood** [comment]
I'm also not sure about


GO:0005874 | microtubule | IBA with PTN000172598 , Q9BVA1 , WBGene00006531 , CAL0000176403 , P24633 , locus:2076750 , Q13509 , MGI:101834 , FBgn0003884 , MGI:107804 , DDB_G0269196 , Q9BUF5 , Q71U36 , DDB_G0287689 , atb2 , WBGene00006536 , P68366 , locus:3439219 , MGI:107812 , FBgn0284243 , A0A1D8PTV4 , P04350 , FBgn0003886 , MGI:107814 , FBgn0087040 , P68363 , P10653 , MGI:1095406 , P68371 , FBgn0260639 , WBGene00003175 , locus:2166469 , WBGene00006538 , MGI:98869 , Q13885 , MGI:2144208 , P23258 , WBGene00006537 , nda3 , MGI:1095409 , Q9NRH3 , P07437 , Q3ZCM7 , FBgn0052396
-- | -- | --


[GO:0005874](https://www.pombase.org/term/GO:0005874)	
[microtubule](https://www.pombase.org/term/GO:0005874)
[IBA](http://wiki.geneontology.org/index.php/Inferred_from_Biological_aspect_of_Ancestor_(IBA)) with [PTN000172598](http://www.pantherdb.org/panther/lookupId.jsp?id=PTN000172598) , [Q9BVA1](https://www.uniprot.org/uniprot/Q9BVA1) , [WBGene00006531](http://www.wormbase.org/get?name=WBGene00006531) , [CAL0000176403](http://www.candidagenome.org/cgi-bin/locus.pl?dbid=CAL0000176403) , [P24633](https://www.uniprot.org/uniprot/P24633) , [locus:2076750](http://arabidopsis.org/servlets/TairObject?accession=locus:2076750) , [Q13509](https://www.uniprot.org/uniprot/Q13509) , [MGI:101834](http://www.informatics.jax.org/accession/MGI:101834) , [FBgn0003884](http://flybase.org/reports/FBgn0003884.html) , [MGI:107804](http://www.informatics.jax.org/accession/MGI:107804) , [DDB_G0269196](http://dictybase.org/gene/DDB_G0269196) , [Q9BUF5](https://www.uniprot.org/uniprot/Q9BUF5) , [Q71U36](https://www.uniprot.org/uniprot/Q71U36) , [DDB_G0287689](http://dictybase.org/gene/DDB_G0287689) , [atb2](https://www.pombase.org/gene/SPBC800.05c) , [WBGene00006536](http://www.wormbase.org/get?name=WBGene00006536) , [P68366](https://www.uniprot.org/uniprot/P68366) , [locus:3439219](http://arabidopsis.org/servlets/TairObject?accession=locus:3439219) , [MGI:107812](http://www.informatics.jax.org/accession/MGI:107812) , [FBgn0284243](http://flybase.org/reports/FBgn0284243.html) , [A0A1D8PTV4](https://www.uniprot.org/uniprot/A0A1D8PTV4) , [P04350](https://www.uniprot.org/uniprot/P04350) , [FBgn0003886](http://flybase.org/reports/FBgn0003886.html) , [MGI:107814](http://www.informatics.jax.org/accession/MGI:107814) , [FBgn0087040](http://flybase.org/reports/FBgn0087040.html) , [P68363](https://www.uniprot.org/uniprot/P68363) , [P10653](https://www.uniprot.org/uniprot/P10653) , [MGI:1095406](http://www.informatics.jax.org/accession/MGI:1095406) , [P68371](https://www.uniprot.org/uniprot/P68371) , [FBgn0260639](http://flybase.org/reports/FBgn0260639.html) , [WBGene00003175](http://www.wormbase.org/get?name=WBGene00003175) , [locus:2166469](http://arabidopsis.org/servlets/TairObject?accession=locus:2166469) , [WBGene00006538](http://www.wormbase.org/get?name=WBGene00006538) , [MGI:98869](http://www.informatics.jax.org/accession/MGI:98869) , [Q13885](https://www.uniprot.org/uniprot/Q13885) , [MGI:2144208](http://www.informatics.jax.org/accession/MGI:2144208) , [P23258](https://www.uniprot.org/uniprot/P23258) , [WBGene00006537](http://www.wormbase.org/get?name=WBGene00006537) , [nda3](https://www.pombase.org/gene/SPBC26H8.07c) , [MGI:1095409](http://www.informatics.jax.org/accession/MGI:1095409) , [Q9NRH3](https://www.uniprot.org/uniprot/Q9NRH3) , [P07437](https://www.uniprot.org/uniprot/P07437) , [Q3ZCM7](https://www.uniprot.org/uniprot/Q3ZCM7) , [FBgn0052396](http://flybase.org/reports/FBgn0052396.html)

since gamma tubunlin complex is really part of the MTOC and the microtubule is made by it, it seems odd to say it is part of a microtubule

**Comment by marcfeuermann** [comment]
I agree. Changes made for gamma tubulin.
